packages/ts/file-router/src/runtime/createMenuItems.ts

```diff
@@ -12,6 +12,9 @@ import type { MenuItem, ViewConfig } from '../types.js';
  * @returns A list of menu items.
  */
 export function createMenuItems(vaadinObject = (window as VaadinWindow).Vaadin): readonly MenuItem[] {
+  // @ts-expect-error: esbuild injection
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+  __REGISTER__('createMenuItems', vaadinObject);
```

Comment:
This has no restrictions to run in the production mode, but should be tracked only in the dev mode. 
I propose to use this:
```
if (vaadinObject.developmentMode) {
  __REGISTER__('createMenuItems', vaadinObject);
}
```
and even better to add this check into __REGISTER__ function.

Reply:
Calling the register is only adding an entry to the `Vaadin.registrations`, but AFAIK, those are not gathered nor sent in PROD mode. If that's not the case, then all Hilla TS packages should change since the same call to `__REGISTER__()` is happening in their index upon loading on the browser.